src/helics/apps/helics-broker-server.cpp

```diff
@@ -277,6 +276,11 @@ void terminalFunction (std::vector<std::string> args)
     {
         std::string cmdin;
         std::cout << "\nhelics_broker_server>>";
+        if (cmdin == "quit()")
```

Comment:
Do any of the other commands use parenthesis?

Reply:
no, this was mainly a way to make klockwork happy,  and just in case the app screwed up and you wanted a way to abort.    Really just needed a short circuit path that bypassed the CLI app and quit immediately.      Open to other suggestions.

Reply:
I think for consistency with other commands it should just be "quit".

Reply:
I changed it to "exit" || "q"  and actually made it after the line that gets the command from the user.